scene/main/viewport.cpp

```diff
@@ -1933,7 +1933,7 @@ void Viewport::_gui_input_event(Ref<InputEvent> p_event) {
 					}
 				}
 
-				if (!is_tooltip_shown && over->can_process()) {
+				if (!is_tooltip_shown && over->can_process() && mm->get_relative() != Vector2()) {
```

Comment:
```suggestion
				if (!is_tooltip_shown && over->can_process() && !mm->get_relative().is_zero_approx()) {
```

Reply:
Makes me wonder if it's a good time to reconsider whether resetting the tooltip at the slightest movement is a good idea. Most software I have used only resets it when moving outside of the "control" that caused it.

Reply:
This only resets the timer for initial display of the tooltip. Once the tooltip is displayed, it will only be removed after the mouse moves outside of the relevant Control.

Reply:
Switched to is_zero_approx().